src/video/x11/SDL_x11window.c

```diff
@@ -1109,6 +1109,14 @@ X11_SetWindowResizable(_THIS, SDL_Window * window, SDL_bool resizable)
     X11_XFlush(display);
 }
 
+void
+X11_SetWindowAlwaysOnTop(_THIS, SDL_Window * window, SDL_bool on_top)
+{
+    SDL_WindowData* data = (SDL_WindowData*) window->driverdata;
+
```

Comment:
This probably needs to do something different if the window is mapped or not.

Check [the SetWindowMaximized code](https://github.com/libsdl-org/SDL/blob/de6d290266d1def0eef9df81bf9be41c12a98c61/src/video/x11/SDL_x11window.c#L1194-L1231), which you can mostly copy/paste in here.

Reply:
Done.